routes/account.rb

```diff
@@ -17,13 +17,15 @@ class Clover
         r.post "disconnect" do
           provider, uid = typecast_params.nonempty_str(["provider", "uid"])
           identities = current_account.identities
-          unless identities.length > 1
-            # YYYY: We can allow to disconnect the last omniauth provider if the user has a password
-            # But need https://github.com/jeremyevans/rodauth/pull/461 to be merged first.
+          unless identities.length > (rodauth.has_password? ? 0 : 1)
```

Comment:
```suggestion
          unless identities.length > (rodauth.has_password? && provider != "password") ? 0 : 1)
```

We should not allow deletion of password unless the user has at least one social login.

Reply:
If the user has a password, `unless identities.length > 0`, so it doesn't allow to delete password if the user doesn't have any account identities.

Reply:
Understood.  If you don't have a password, you need at least 2 existing identities, because the assumption is the provider is an an identity provider, because you should not be deleting a password if you do not have a password.